scripts/travis-test.sh

```diff
@@ -27,15 +27,16 @@ log_and_run() {
 }
 
 run_tests() {
-  (cd ${ROOT_DIR} && log_and_run test/run-tests.py ${RUN_TEST_ARGS} $* --timeout=10)
+  (cd ${ROOT_DIR} && log_and_run test/run-tests.py --bindir ${BINDIR} $* --timeout=10) && true
+  (log_and_run ${BINDIR}/wabt-unittests) && true
```

Comment:
Do you need the && true here?

Also do you need the brackes around this line (its not doing a chdir, like the one above).